pangeo-notebook/packages.txt

```diff
@@ -674,65 +704,74 @@ uc-micro-py==1.0.3
 ucx==1.18.0
 ujson==5.10.0
 unicodedata2==16.0.0
+universal_pathlib==0.2.6
 uri-template==1.3.0
 uriparser==0.9.8
 urllib3==1.26.19
 uvicorn==0.34.0
 uvicorn-standard==0.34.0
 uvloop==0.21.0
+virtualizarr==1.2.0
 watchfiles==1.0.4
 watermark==2.5.0
 wayland==1.23.1
-wayland-protocols==1.37
+wayland-protocols==1.41
 wcwidth==0.2.13
 webcolors==24.11.1
 webencodings==0.5.1
 webob==1.8.9
 websocket-client==1.8.0
-websockets==14.2
+websockets==15.0.1
 wheel==0.45.1
 widgetsnbextension==4.0.13
 wrapt==1.17.2
 x264==1!164.3095
 x265==3.5
-xarray==2025.1.1
+xarray==2025.1.2
 xarray-spatial==0.4.0
 xarray_leaflet==0.2.3
 xarrayutils==2.0.1
 xbatcher==0.4.0
-xclim==0.54.0
+xclim==0.55.1
 xerces-c==3.2.5
 xesmf==0.8.8
-xgboost==2.1.3
+xgboost==2.1.4
 xgcm==0.8.1
 xhistogram==0.3.2
+xkeyboard-config==2.43
 xmip==0.7.2
 xmitgcm==0.5.2
 xorg-libice==1.1.2
-xorg-libsm==1.2.5
-xorg-libx11==1.8.10
+xorg-libsm==1.2.6
+xorg-libx11==1.8.12
 xorg-libxau==1.0.12
+xorg-libxcomposite==0.4.6
+xorg-libxcursor==1.2.3
 xorg-libxdamage==1.1.6
 xorg-libxdmcp==1.1.5
 xorg-libxext==1.3.6
 xorg-libxfixes==6.0.1
 xorg-libxi==1.8.2
+xorg-libxinerama==1.1.5
+xorg-libxrandr==1.5.4
 xorg-libxrender==0.9.12
+xorg-libxscrnsaver==1.2.4
+xorg-libxtst==1.2.5
 xorg-libxxf86vm==1.1.6
 xpublish==0.3.3
 xrft==1.0.1
 xskillscore==0.0.26
 xvec==0.3.1
-xxhash==0.8.2
+xxhash==0.8.3
 xyzservices==2025.1.0
 yamale==5.3.0
 yaml==0.2.5
 yarl==1.18.3
-zarr==3.0.1
+zarr==2.18.4
```

Comment:
Something downgrading zarr to v2 again

Reply:
Oo, looks like earthaccess pins to zarr v2 - https://github.com/nsidc/earthaccess/pull/923.

Reply:
Ok, I'm pinning to `earthaccess<0.13.0` and `intake-esm<2025.2.3` to ensure that `zarr>=3` is still being pulled in. That means staying on `earthaccess=0.12.0` and `intake-esm=2024.2.6`.

Reply:
Looks like locking still timed out :( https://github.com/pangeo-data/pangeo-docker-images/actions/runs/14181683891 , maybe pin zarr>=3 too?

Reply:
Nope, still timing out after pinning to `zarr>=3`. I tried locking `pangeo-notebook` on my machine and it took 35 min :sweat_smile:

Reply:
Ugh. Switching to pixi is more and more appealing (https://github.com/pangeo-data/pangeo-docker-images/issues/571). But I'll just bump to 45min on the main branch for the time being

Reply:
Yeah, thanks for bumping the timeout limit. I'd say we give conda-lock v3 a try (just came out with mamba 2 support), and see if it improves things?

Reply:
cool, yeah I also saw they added a migration guide to the docs, but also happy to just keep chugging along with whats working :) https://conda.github.io/conda-lock/pixi-migration/